src/server/page_server.hpp

```diff
@@ -249,7 +249,7 @@ class page_server
     pts_mvcc_tracker m_pts_mvcc_tracker;
 };
 
-extern page_server ps_Gl;
+extern std::unique_ptr<page_server> ps_Gl;
```

Comment:
Could we move this in `server_type.h` - similar to where `ts_Gl` is?
The include clauses would be leaner.

Reply:
Thank you, I missed it.
It's moved in server_type.hpp/cpp inside SERVER_MODE.